backend/ventserver/integration/_trio.py

```diff
@@ -312,3 +334,49 @@ async def process_all(
             nursery.start_soon(
                 process_clock, protocol, channel, push_endpoint.clone()
             )
+
+
+async def initialize_states(
+        states: List[Type[betterproto.Message]],
+        protocol: server.Protocol,
+        filehandler: fileio.Handler,
+        all_states: Dict[
+            Type[betterproto.Message], Optional[betterproto.Message]
+        ]
+) -> None:
+    """Initialize state values from state store or default values."""
+    default_init = list()
+    for state in states:
+        try: # Handle fileio errors
+            filehandler.set_props(state.__name__, "rb")
+            await filehandler.open()
+            message = await filehandler.read()
+            protocol.receive.file.input(
+                file.StateData(state_type=state.__name__, data=message)
+                )
+        except OSError as err:
+            print(err)
+        finally:
+            await filehandler.close()
+
+    while True:
+        try: # Handles data integrity and protocol error
+            event = protocol.receive.file.output()
+            if not event:
+                break
+
+            all_states[type(event)] = event
```

Comment:
Would it be feasible to have the server protocol or the backend protocol do the work of filling in the value in `all_states` (e,g, maybe even just using the backend's `file_state_synchronizer` to do that work), instead of doing it in `integration._trio`?

Reply:
`initialize_states()` was in `application.py` and `simulation.py` but since `pylint` was throwing duplicated code warnings, I had to move here in `_trio.py`. Otherwise, it is being called at the startup in `application.py` and `simulation.py`. 
In this case `all_states` is being initialized at the startup, I'm not sure how `file_state_synchronizer` can be used to fill in `all_states`.  May be I'll take a look at it and see what can be done.